site/content/docs/developer-guide/custom-indexes.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Hosting Custom Plugin Indexes
+slug: custom-indexes
+weight: 500
```

Comment:
```suggestion
weight: 900
```
To me this sounds like a much more specialized topic than using custom indexes. So I would put it last.

Reply:
I think the weight only applies to the folder that it lives in. I added it here in the developer guide above the plugin usage stats page.